channeldb/channel.go

```diff
@@ -822,6 +825,10 @@ type OpenChannel struct {
 	// default ShortChannelID. This is only set for zero-conf channels.
 	confirmedScid lnwire.ShortChannelID
 
+	// sentShutdown denotes whether or not we've sent the Shutdown message
+	// for this channel.
+	sentShutdown bool
```

Comment:
do we need to do a migration for `historicalChanBucket`?

Reply:
shouldn't need to since this is in a tlv stream